EasyPost/Services/UserService.cs

```diff
@@ -40,26 +41,26 @@ public async Task<User> CreateChild(Dictionary<string, object> parameters)
         /// <param name="parameters"><see cref="BetaFeatures.Parameters.Users.CreateChild"/> parameter set.</param>
         /// <returns><see cref="User"/> instance.</returns>
         [CrudOperations.Create]
-        public async Task<User> CreateChild(BetaFeatures.Parameters.Users.CreateChild parameters)
+        public async Task<User> CreateChild(BetaFeatures.Parameters.Users.CreateChild parameters, CancellationToken cancellationToken = default)
         {
             // Because the normal CreateChild method does wrapping internally, we can't simply pass the parameters object to it, otherwise it will wrap the parameters twice.
-            return await RequestAsync<User>(Method.Post, "users", parameters.ToDictionary());
+            return await RequestAsync<User>(Method.Post, "users", cancellationToken, parameters.ToDictionary());
         }
 
         /// <summary>
-        ///     Retrieve a User from its id. If no id is specified, it returns the user for the api_key specified.
```

Comment:
issue: I don't think we want to remove this functionality. Users have come to rely on it and our docs clearly state this is a reasonable thing to do.

Reply:
Fair enough, reverting.